quickstart.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
```

Comment:
Should we remove this file or we can replace it with new quickstart-v5.0.2.sh
Thoughts?